habushu-mixology-consumer/tests/features/generate-report-with-manual-tests.feature

```diff
@@ -0,0 +1,9 @@
+@manual
```

Comment:
Could we not automate this test? if all were doing is checking to see if an HTML report is in the target directory, this should be fairly easy to do.

Reply:
I think the point is to have a test that doesn't generate anything to simulate missing tests in the wild?

Reply:
@d-ryan-ashcraft is correct. It's a bit of a weird setup, but it's effectively a means of reproducing the issue. That said, since this is specifically testing the issue, I can see the argument for removing it.